patrace/project/cmake/toolchains/wayland_aarch64.cmake

```diff
@@ -1,9 +1,19 @@
 SET(CMAKE_SYSTEM_NAME Linux)
 
-SET(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
-SET(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
+if (NOT DEFINED ENV{CROSS_COMPILE})
+    SET(ENV{CROSS_COMPILE} aarch64-linux-gnu-)
+endif()
+
+SET(CMAKE_C_COMPILER $ENV{CROSS_COMPILE}gcc)
+SET(CMAKE_CXX_COMPILER $ENV{CROSS_COMPILE}g++)
 SET(WINDOWSYSTEM wayland)
 SET(ARCH aarch64)
 
-set(CC_HOST "aarch64-linux-gnu")
-set(ENV{PKG_CONFIG_PATH} "/usr/lib/aarch64-linux-gnu/pkgconfig")
+if (DEFINED ENV{SDKTARGETSYSROOT})
+    SET(CMAKE_C_FLAGS "--sysroot=$ENV{SDKTARGETSYSROOT} ")
+    SET(CMAKE_CXX_FLAGS "--sysroot=$ENV{SDKTARGETSYSROOT} ")
+    message(STATUS "ENV{SDKTARGETSYSROOT}=" $ENV{SDKTARGETSYSROOT})
+else()
+    set(CC_HOST  "aarch64-linux-gnu")
+    set(ENV{PKG_CONFIG_PATH} "./scripts/pkgconfig/aarch64-linux-gnu")
```

Comment:
This does not seem to exist?

Reply:
the default CC variable in the Yocto toolchain has --sysroot options compared to the CC variable in this file, so this option is added in the CMAKE_C_FLAGS and CMAKE_CXX_FLAGS, otherwise there will be some link issue.

CC_HOST and PKG_CONFIG_PATH were used before, i delete this and build with gcc-arm-8.3-2019.03-x86_64-aarch64-linux-gnu, wayland_aarch64 can be built successfully, but i can't whether there is any runtime error. this can be deleted if you can confirm it.

Reply:
I mean, the file"./scripts/pkgconfig/aarch64-linux-gnu" does not exist in the repo and is not added here, so where does it come from?

Reply:
hi,  this difference was introduced by the rebase, and the commit has been updated.

Reply:
ping, @per-mathisen-arm , can you merge this change in the next?